interestingness/crashes.py

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env python
+# coding=utf-8
+# pylint: disable=invalid-name,missing-docstring
 
 from __future__ import print_function
-from optparse import OptionParser
+from optparse import OptionParser  # pylint: disable=deprecated-module
 
-import timedRun
+import timedRun  # pylint: disable=relative-import
```

Comment:
These are probably better fixed instead of disabled, by making it `from . import timedRun` ... which _might_ require `from __future__ import absolute_import` ?